Makefile.shared

```diff
@@ -80,14 +82,25 @@ push-image:
 # helm charts
 ###
 lint-chart-default:
+	mkdir -p ${CHART_DIR}
```

Comment:
Could you please create a standalone Make target for compiling charts (for example `build-chart`), similar to what we have for images? This would be helpful for local deployments, especially when you want to render a chart without pushing it to the registry.